cli/azd/pkg/project/project.go

```diff
@@ -82,6 +82,12 @@ func Parse(ctx context.Context, yamlContent string) (*ProjectConfig, error) {
 		projectConfig.Infra.Path = "infra"
 	}
 
+	if strings.Contains(projectConfig.Infra.Path, "\\") && !strings.Contains(projectConfig.Infra.Path, "/") {
```

Comment:
In an earlier iteration I had tried creating a new type called `osutil.PathString` which had a `String()` method and then used that on `ProjectConfig` and friends - but it started to feel pretty tedious.

It might be better to handle this with custom unmarshalling logic at the ProjectConfig and nested type layers - where they can apply the normalization, not sure.  It is nice that everything is shunted through these save and load methods.